runtime/compiler/optimizer/J9TransformUtil.cpp

```diff
@@ -139,7 +139,7 @@ static bool isFinalFieldPointingAtRepresentableNativeStruct(TR::SymbolReference
       case TR::SymbolReferenceTable::classFromJavaLangClassSymbol:
       case TR::SymbolReferenceTable::classFromJavaLangClassAsPrimitiveSymbol:
          // Note: We could also do vftSymbol, except replacing those with
-         // loadaddres mucks up indirect loads in ways the optimizer/codegen
+         // loadaddress mucks up indirect loads in ways the optimizer/codegen
```

Comment:
@andrewcraik is this better as `load address`?

Reply:
this would be better as `a loadaddr` where loadaddr is the name of an opcode and is what is being referred to here.